packages/release/release.spec

```diff
@@ -229,11 +243,33 @@ install -p -m 0644 %{S:1100} \
 install -d %{buildroot}%{_cross_unitdir}/systemd-resolved.service.d
 install -p -m 0644 %{S:1101} \
   %{buildroot}%{_cross_unitdir}/systemd-resolved.service.d/00-env.conf
+install -p -m 0644 %{S:1106} \
+  %{buildroot}%{_cross_unitdir}/systemd-resolved.service.d/10-private-tmp.conf
 
 install -d %{buildroot}%{_cross_unitdir}/systemd-networkd.service.d
 install -p -m 0644 %{S:1102} \
   %{buildroot}%{_cross_unitdir}/systemd-networkd.service.d/00-env.conf
 
+install -d %{buildroot}%{_cross_unitdir}/systemd-udev-trigger.service.d/
+install -p -m 0644 %{S:1105} \
+  %{buildroot}%{_cross_unitdir}/systemd-udev-trigger.service.d/00-selinux.conf
+
+install -d %{buildroot}%{_cross_unitdir}/systemd-sysusers.service.d
+install -p -m 0644 %{S:1107} \
+  %{buildroot}%{_cross_unitdir}/systemd-sysusers.service.d/50-systemd-sysusers.conf
```

Comment:
`%{S:1107}` is `issue` - this would be `%{S:1109}` unless it's renumbered after `issue` moves.